chef/cookbooks/resolver/recipes/default.rb

```diff
@@ -57,6 +57,7 @@
         # invalidate dnsmasq cache if local zone changes
         subscribes :reload, "template[/etc/bind/db.#{node[:dns][:domain]}]"
       end
+      not_if { ::File.exist?("/var/lib/crowbar/install/disable_dns") }
```

Comment:
This should be the same condition as in the other recipe: `node["crowbar"]["admin_node"] && ::File.exist?("/var/lib/crowbar/install/disable_dns")`